mailpoet/assets/css/src/components-automation-editor/_add-step-button.scss

```diff
@@ -0,0 +1,14 @@
+.mailpoet-automation-editor-add-step-button {
```

Comment:
regarding naming: how `mailpoet-` specific should it be?

Reply:
I've discussed this dilemma with Tautvidas and the decision was to prefix hooks and stuff with `mailpoet` for now. Seems like the requirement is "being able to extract it and make reusable quickly" rather than "having it completely reusable immediately". We don't have a more universal name at the moment (apart from "automation" maybe) so I guess we can keep the prefixes now (or omit the `mailpoet-` part).